.github/workflows/ci.yml

```
@@ -18,72 +18,8 @@ permissions:
-    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
-
-    # Definition of the build matrix
-    strategy:
-      matrix:
-        java: [11, 17]
-        entry:
-          - { mock-maker: 'mock-maker-default', member-accessor: 'member-accessor-default' }
-          - { mock-maker: 'mock-maker-inline', member-accessor: 'member-accessor-module' }
-          - { mock-maker: 'mock-maker-subclass', member-accessor: 'member-accessor-module' }
-          - { mock-maker: 'mock-maker-subclass', member-accessor: 'member-accessor-reflection' }
-          - { mock-maker: 'mock-maker-inline', member-accessor: 'member-accessor-reflection' }
-
-    # All build steps
-    # SINGLE-MATRIX-JOB means that the step does not need to be executed on every job in the matrix
-    steps:
-
-    - name: 1. Check out code
-      uses: actions/checkout@v3 # https://github.com/actions/checkout
-      with:
-        fetch-depth: '0' # https://github.com/shipkit/shipkit-changelog#fetch-depth-on-ci
-
-    - name: 2. Set up Java ${{ matrix.java }}
-      uses: actions/setup-java@v3
-      with:
-        distribution: 'zulu'
-        java-version: ${{ matrix.java }}
-
-    - name: 3. Validate Gradle wrapper
-      if: matrix.java == 11 && matrix.entry.mock-maker == 'mock-maker-default' # SINGLE-MATRIX-JOB
-      uses: gradle/wrapper-validation-action@v1.0.6 # https://github.com/gradle/wrapper-validation-action
-
-    - name: 4. Build and check reproducibility of artifacts (single job only)
-      if: matrix.java == 11 && matrix.entry.mock-maker == 'mock-maker-default' # SINGLE-MATRIX-JOB
-      run: ./check_reproducibility.sh
-
-    - name: 5. Spotless check (single job only). Run './gradlew spotlessApply' locally if this job fails.
-      if: matrix.java == 11 && matrix.entry.mock-maker == 'mock-maker-default' # SINGLE-MATRIX-JOB
-      run: ./gradlew spotlessCheck
-
-    - name: 6. Build on Java ${{ matrix.java }} with ${{ matrix.entry.mock-maker }} and ${{ matrix.entry.member-accessor }}
-      run: ./gradlew build idea --scan
-      env:
-        MOCK_MAKER: ${{ matrix.entry.mock-maker }}
-        MEMBER_ACCESSOR: ${{ matrix.entry.member-accessor }}
-
-    - name: 7. Upload coverage report
-      run: |
-        ./gradlew coverageReport -s --scan && cp build/reports/jacoco/mockitoCoverage/mockitoCoverage.xml jacoco.xml
-        curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import # One-time step
-
-        curl -Os https://uploader.codecov.io/latest/linux/codecov
-        curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM
-        curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM.sig
-
-        gpgv codecov.SHA256SUM.sig codecov.SHA256SUM
-        shasum -a 256 -c codecov.SHA256SUM
-
-        chmod +x codecov
-        ./codecov
+  build-and-test:
+   uses: mockito/mockito/.github/workflows/build-and-test.yml@feature/build-once-test-all
```

This might be why it's not runnable: this is referencing a potentially non-existent file because this branch exists only in your fork?

I would recommend using the [relative notation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iduses):
```suggestion
  build-and-test:
   uses: ./.github/workflows/build-and-test.yml
```

---

Let's resolve this too.